README.md

```diff
@@ -37,36 +37,27 @@ First, create a container for `traefik-pages`:
       - SITES_ROOT=/mnt/sites
       - TRAEFIK_SERVICE=traefik-pages@docker
       - AUTH_PASSWORD=hunter2
+    labels:
+      - traefik.enable=true
+      - traefik.http.services.traefik-pages.loadbalancer.server.port=5000
```

Comment:
This line shouldn't be necessary, as Traefik will identify the port automatically through the `EXPOSE` in the `Dockerfile`.